### ISSUE TITLE
Exception 'Error' when calling run-pending-sendouts from console 

### ISSUE THREAD

**Issue by Saboteur777** [putyourlightson/craft-campaign#45]
I get an error when calling the `run-pending-sendouts` task: 

**php craft campaign/sendouts/run-pending-sendouts**
```
0 pending sendout(s) queued.
Exception 'Error' with message 'Class 'putyourlightson\campaign\console\controllers\ExitCode' not found'

in /var/www/vendor/putyourlightson/craft-campaign/src/console/controllers/SendoutsController.php:40

Stack trace:
#0 [internal function]: putyourlightson\campaign\console\controllers\SendoutsController->actionRunPendingSendouts()
#1 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#2 /var/www/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#3 /var/www/vendor/yiisoft/yii2/console/Controller.php(148): yii\base\Controller->runAction('run-pending-sen...', Array)
#4 /var/www/vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('run-pending-sen...', Array)
#5 /var/www/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('campaign/sendou...', Array)
#6 /var/www/vendor/yiisoft/yii2/console/Application.php(147): yii\console\Application->runAction('campaign/sendou...', Array)
#7 /var/www/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
#8 /var/www/craft(22): yii\base\Application->run()
#9 {main}
```

**php -v**
```
PHP 7.2.12 (cli) (built: Nov 28 2018 22:58:16) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.12, Copyright (c) 1999-2018, by Zend Technologies
```

**Comment by putyourlightson** [comment]
Thanks, this was fixed in version 1.5.5.

**Comment by Saboteur777** [comment]
It works now, thanks!
